k8-operator/controllers/infisicalpushsecret/infisicalpushsecret_helper.go

```
@@ -440,9 +486,28 @@ func (r *InfisicalPushSecretReconciler) DeleteManagedSecrets(ctx context.Context
 
 	resourceVariables := r.getResourceVariables(infisicalPushSecret)
 	infisicalClient := resourceVariables.InfisicalClient
+	cancelCtx := resourceVariables.CancelCtx
+	authDetails := resourceVariables.AuthDetails
+	var err error
+
+	if authDetails.AuthStrategy == "" {
+		logger.Info("No authentication strategy found. Attempting to authenticate")
+		authDetails, err = r.handleAuthentication(ctx, infisicalPushSecret, infisicalClient)
+		r.SetAuthenticatedStatusCondition(ctx, &infisicalPushSecret, err)
```

golangci-lint (1.64.8)

Error return value of `r.SetAuthenticatedStatusCondition` is not checked

(errcheck)